src/main/java/com/epam/tc/hw7/AbstractTest.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.tc.hw7;
+
+import static com.epam.jdi.light.driver.WebDriverUtils.killAllSeleniumDrivers;
+import static com.epam.jdi.light.elements.init.PageFactory.initSite;
+
+import com.epam.tc.hw7.site.Site;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.BeforeTest;
+
+public abstract class AbstractTest {
+    @BeforeTest(alwaysRun = true)
+    public void setUp() {
+        initSite(Site.class);
```

Comment:
It was mentioned that you need to initSite only once per Suite, as it's a very difficult operation